yetanotherco/aligned_layer
# fix: swap variables declaration in BatcherPaymentService.sol

# Description
The order of these variables is incorrect
```solidity
    // STORAGE
    address public batcherWallet;

    IAlignedLayerServiceManager public alignedLayerServiceManager;
```

The correct order is 
```solidity
    // STORAGE
    IAlignedLayerServiceManager public alignedLayerServiceManager;

    address public batcherWallet;
```

# How to test
1. Run anvil
`make anvil_start_with_block_time`
2. Run aggregator
`make aggregator_start`
3. Run batcher
`make batcher_start_local`
4. Run operator
`export OPERATOR_ADDRESS=0x70997970C51812dc3A010C7d01b50e0d17dc79C8`
`make operator_full_registration CONFIG_FILE=config-files/config-operator-1.yaml`
`make operator_start CONFIG_FILE=config-files/config-operator-1.yaml`

5. Send proofs and check the aggregator is verifying the batches

Also, to make sure addresses have not swapped, you can:
```
> cast call 0x7969c5eD335650692Bc04293B07F5BF2e7A673C0 "batcherWallet()(address)"
> 0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc
```
```
> cast call 0x7969c5eD335650692Bc04293B07F5BF2e7A673C0 "alignedLayerServiceManager()(address)"
> 0x1613beB3B2C4f22Ee086B2b38C1476A3cE7f78E8
```
